examples/storage/couchdb/images/readiness_probe/join_cluster.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+"""
+    Usage:  join_cluster.py dbname dbname ...
+"""
+from subprocess import check_output, CalledProcessError
+import requests
+import logging
+import socket
+import json
+import sys
+import os
+
+
+def main(system_dbs):
+    pod_hostnames = discover_pod_fq_hostnames()
+    logging.warning('Using the discovered host names: {0}'.format(pod_hostnames))
+
+    cluster_admin = (os.environ['CLUSTER_ADM_USER'].rstrip(), os.environ['CLUSTER_ADM_PASS'].rstrip())
+    populate_nodes_db(pod_hostnames, cluster_admin)
+    create_system_dbs(cluster_admin, system_dbs)
+    sys.exit(0)
+
+
+def discover_pod_fq_hostnames():
```

Comment:
The petset docs recommend doing the service discovery using a DNS lookup so I just followed that.  I shall give that python module ago.